src/pages/learn/digital-gardening/index.tsx

```diff
@@ -122,12 +122,25 @@ export const digitalGardeningQuery = groq`*[_type == 'resource' && slug.current
   'cta': content[title == 'cta'][0]{
     description
   },
-  resources[]{
+  'featured': resources[title == 'Featured digital gardening courses'][0]{
+ 		'courses': resources[]{
+    	title,
+    	byline,
+      'name': content[title == 'name'][0].description,
+    	'path': resources[]->[0].path,
+    	'image': resources[]->[0].image
+  	}
+  },
+	'talks': resources[title == 'Infrastructure for Digital Gardens'][0]{
```

Comment:
moved the talks up to the resources section (previously in related resources) and now filtering by resource name

Reply:
ya this is better imo. i think query by title is brittle and we should use _IDs or slugs